eclair-core/eclair-cli

```diff
@@ -31,13 +32,21 @@ Full documentation at: <https://github.com/ACINQ/eclair>"
 
 # Executes a JSON RPC call to a node listening on ${URL}
 call() {
-    jqexp='if .error == null then .result else .error.message end'
+    jqexp='.'   # 'if .error == null then .result else .error.message end'
     # override default jq parsing expression
-    if [ $# -ge 3 ] && [ ${FULL_OUTPUT} == "false" ]; then jqexp=${3}; fi
+    # if [ $# -ge 3 ] && [ ${FULL_OUTPUT} == "false" ]; then jqexp=${3}; fi
+
     # set password
     if [ -z ${PASSWORD} ]; then auth="eclair-cli";
     else auth="eclair-cli:"${PASSWORD}; fi
-    eval curl "--user ${auth} --silent --show-error -X POST -H \"Content-Type: application/json\" -d '{ \"method\": \"'${1}'\", \"params\": '${2}' }' ${URL}" | jq -r "$jqexp"
+    # collect form data from 2nd parameter
+    form_data=""
+    for param in ${2}; do
+      form_data="$form_data -F \"$param\""
+    done;
+    #echo "FORM_DATA: $form_data"
```

Comment:
`#echo "FORM_DATA: $form_data"` should be cleaned up

Reply:
Done via 72cf2e8